yarn-project/boxes/private-token/package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "private-token",
+  "private": true,
+  "version": "0.1.0",
+  "type": "module",
+  "main": "./dest/index.js",
+  "scripts": {
+    "build": "yarn clean && webpack",
+    "install:noir": "curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash noirup -v aztec",
+    "install:sandbox": "docker pull aztecprotocol/aztec-sandbox:latest",
+    "clean": "rm -rf ./dest .tsbuildinfo",
+    "start": "serve -p 3000 ./dest",
+    "start:dev": "webpack serve --mode=development",
+    "start:sandbox": "/bin/bash -c \"$(curl -fsSL 'https://sandbox.aztec.network')\" ",
+    "formatting": "prettier --check ./src && eslint ./src",
+    "formatting:fix": "prettier -w ./src",
+    "compile": "aztec-compile compile src/contracts --outdir ../artifacts --typescript ../artifacts",
+    "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --runInBand",
+    "test:integration": "concurrently -k -s first -c reset,dim -n test,anvil \"yarn test\" \"anvil\""
+  },
+  "jest": {
+    "preset": "ts-jest/presets/default-esm",
+    "globals": {
+      "ts-jest": {
+        "useESM": true
+      }
+    },
+    "transform": {
+      "^.+\\.(ts|tsx)$": "ts-jest"
+    },
+    "moduleNameMapper": {
+      "^(\\.{1,2}/.*)\\.js$": "$1"
+    },
+    "testRegex": "./src/.*\\.test\\.ts$",
+    "rootDir": "./src"
+  },
+  "dependencies": {
+    "@aztec/aztec.js": "workspace:^",
+    "@aztec/circuits.js": "workspace:^",
+    "@aztec/cli": "workspace:^",
+    "@aztec/foundation": "workspace:^",
+    "@aztec/noir-compiler": "workspace:^",
+    "@aztec/types": "workspace:^",
```

Comment:
Can also remove `@aztec/types`. Users should be able to import types they need to use aztec.js directly from `aztec.js`.